lib/license.js

```diff
@@ -100,57 +106,33 @@ function fixCli(pkgPath) {
       case '(Artistic-2.0 OR LicenseRef-LICENSE.md)': // OBSOLETE
```

Comment:
These `case` statements are fairly repetitive and I can see why they have to be. Might be more clear this way. Have you considered other strategies like creating mappings and using regex's to match any input given to the actual mapping?

`Artistic-2.0 OR LicenseRef-LICENSE -> licenseMappings['Artistic-2.0']()`